packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemFileHandle.kt

```diff
@@ -0,0 +1,49 @@
+package expo.modules.filesystem.next
+
+import expo.modules.kotlin.sharedobjects.SharedObject
+import java.nio.ByteBuffer
+import java.nio.channels.FileChannel
+class FileSystemFileHandle(file: FileSystemFile) : SharedObject() {
```

Comment:
```suggestion

class FileSystemFileHandle(file: FileSystemFile) : SharedObject(), AutoCloseable {
```
It probably doesn't matter that much, but I would use the `AutoCloseable` interface.